packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -121,6 +122,41 @@ - (void)testHLSControls {
   XCTAssertEqualWithAccuracy([videoInitialization[@"duration"] intValue], 4000, 200);
 }
 
+- (void)testTransformFix {
+  NSObject<FlutterPluginRegistry> *registry =
+      (NSObject<FlutterPluginRegistry> *)[[UIApplication sharedApplication] delegate];
+  NSObject<FlutterPluginRegistrar> *registrar = [registry registrarForPlugin:@"TestTransformFix"];
+  FLTVideoPlayerPlugin *videoPlayerPlugin =
+      (FLTVideoPlayerPlugin *)[[FLTVideoPlayerPlugin alloc] initWithRegistrar:registrar];
+
+  FlutterError *error;
+  [videoPlayerPlugin initialize:&error];
+  XCTAssertNil(error);
+
+  FLTCreateMessage *create = [FLTCreateMessage
+      makeWithAsset:nil
+                uri:@"https://flutter.github.io/assets-for-api-docs/assets/videos/bee.mp4"
```

Comment:
We should probably extract the fixup method to a standalone utility method that we can test directly (with mock `AVAssetTrack`s). We shouldn't be using actual videos for each test here as that will be very hard to maintain.

Reply:
The reason why I wrote them here is most of the other tests didn't seem to require the initialization explicitly. I'll extract them later.

Reply:
I'm not sure how that responds to my comment. This PR is changing the transforms, and we should set up the tests—for this PR—to allow testing the transforms without having to add new videos.

Reply:
Not sure about "without having to add new videos", since no videos have been added here.

Reply:
Oh, sure. I miss reading something. :( I'll take some thoughts about your idea and see if I can come up with new implementations.

Reply:
I split this out with a new category (AVAssetTrack+Utils), also added the `FakeAVAssetTrack` in the test.